### DISCUSSION TITLE
Add Button '000'  to Initial MIDI

### DISCUSSION THREAD

**Discussion by thbeckpr** [GrandOrgue/grandorgue#2063] [Developers]
I would like to request that the 000 button on the Combination Setter panel be added to the Initial MIDI sequencer group in settings.

My combination controller app uses the Combination Setter buttons to activate a combination but first it must zero out any existing combination. It then 'taps' the appropriate buttons to activate the desired combination. So the first button 'tapped' must be 000.

Thanks

**Comment by kerkovits** [comment]
This has been requested a plenty of times.
https://github.com/GrandOrgue/grandorgue/discussions/1081
and
https://github.com/GrandOrgue/grandorgue/issues/1974

It seems that @oleg68 is working on it, as it is added to the planned features of the next milestone.
